### ISSUE TITLE
TypeError jQuery.$ is not a function

### ISSUE THREAD

**Issue by BenDol** [workingflows/gwt-jquery#1]
Any ideas why I'm getting this error message?

```
SEVERE: (TypeError) : ((intermediate value)(intermediate value) , $wnd).jQuery.$ is not a function
com.google.gwt.core.client.JavaScriptException: (TypeError) : ((intermediate value)(intermediate value) , $wnd).jQuery.$ is not a function.
```

Making the call like so:

``` java
JQueryElement.$("#jsinterop");
```

Thanks!

**Comment by BenDol** [comment]
Using 2.8.0-SNAPSHOT

**Comment by csrinaldi** [comment]
You have a JQuery script inyected in head section?

**Comment by BenDol** [comment]
Yep it is injected, I even added it to a `RepeatingCommand` until jQuery is injected then it executes the code and still getting this error.

**Comment by csrinaldi** [comment]
I will try to replicate the error and fix it

**Comment by BenDol** [comment]
Cool, any luck reproducing this issue?

**Comment by BenDol** [comment]
Still unable to get this working, guessing the API is outdated with 2.8.0-SNAPSHOT?

**Comment by BenDol** [comment]
Resolved in PR #2
